pkg/controller/replication/conversion.go

```diff
@@ -247,6 +248,14 @@ func (c conversionClient) Patch(ctx context.Context, name string, pt types.Patch
 	return nil, errors.New("Patch() is not implemented for conversionClient")
 }
 
+func (c conversionClient) Apply(ctx context.Context, rs *appsv1apply.ReplicaSetApplyConfiguration, opts metav1.ApplyOptions) (*apps.ReplicaSet, error) {
+	return nil, errors.New("Patch() is not implemented for conversionClient")
```

Comment:
nit: Even though Apply uses Patch under the hood, these should technically be Apply() and ApplyStatus() right?

Reply:
Good catch.